.gitignore

```diff
@@ -17,10 +17,12 @@ data/tmp
 
 # SQLite3 default db location and after migration
 data/*.db
+data/foglamp.db-journal
```

Comment:
As sqlite file name can set via DEFAULT_SQLITE_DB_FILE, hence please make it `data/*.db-journal`